kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/interpreter/api/KuronometerApiClientConfig.scala

```diff
@@ -0,0 +1,10 @@
+package com.github.pedrovgs.kuronometer.free.interpreter.api
+
+case class KuronometerApiClientConfig(
```

Comment:
Right now this class only holds constants. I've used this ``Config`` case class to provide a context around the constant values but configure it in runtime using any config file is not needed. I think we can keep it as simple as is right now 😄